### ISSUE TITLE
"triton vlan create" should not assert on missing argument

### ISSUE THREAD

**Issue by jclulow** [joyent/node-triton#257]
It would appear that `triton vlan create` will explode on an assertion if the `--name` flag is not provided; e.g.,

```
$ triton vlan create 41 
/home/jclulow/.node/node_modules/triton/node_modules/assert-plus/assert.js:22
    throw new assert.AssertionError({
    ^

AssertionError [ERR_ASSERTION]: opts.name (string) is required
    at CloudApi.createFabricVlan (/home/jclulow/.node/node_modules/triton/lib/cloudapi2.js:606:12)
    at onSetup (/home/jclulow/.node/node_modules/triton/lib/do_vlan/do_create.js:59:18)
    at /home/jclulow/.node/node_modules/triton/lib/common.js:888:13
    at promptPassphraseUnlockKey (/home/jclulow/.node/node_modules/triton/lib/common.js:823:9)
    at /home/jclulow/.node/node_modules/triton/lib/common.js:885:9
    at /home/jclulow/.node/node_modules/triton/lib/tritonapi.js:526:13
    at /home/jclulow/.node/node_modules/triton/node_modules/smartdc-auth/lib/keyring.js:193:3
    at /home/jclulow/.node/node_modules/triton/node_modules/smartdc-auth/lib/keyring.js:148:3
    at /home/jclulow/.node/node_modules/triton/node_modules/smartdc-auth/node_modules/vasync/lib/vasync.js:83:5
    at /home/jclulow/.node/node_modules/triton/node_modules/smartdc-auth/lib/keyring.js:159:4
```

This should result in a usage error, not a diagnostic.

**Comment by jclulow** [comment]
Change for review @ https://cr.joyent.us/4763

**Comment by jclulow** [comment]
New usage information from `triton vlan create --help`:

```
Create a VLAN.

Usage:
    triton vlan create [OPTIONS] VLAN

Options:
    -h, --help  Show this help.

  Create options:
    -n NAME, --name=NAME
                Name of the VLAN.
    -D DESC, --description=DESC
                Description of the VLAN.

  Other options:
    -j, --json  JSON stream output.

Example:
    triton vlan create -n "dmz" -D "Demilitarized zone" 73
```

**Comment by jclulow** [comment]
## Testing Notes

In addition to checking the updated `triton vlan create --help` output (above), I have been using these changes to manipulate VLANs in our staging environment for some time now.

**Comment by jclulow** [comment]
Published to NPM registry as `triton@6.2.0`.
